samples/XCT.Sample/Pages/Views/Popups/PopupSize.cs

```diff
@@ -4,7 +4,7 @@ namespace Xamarin.CommunityToolkit.Sample.Pages.Views.Popups
 {
 	static class PopupSize
 	{
-		public static Size Android => new Size(1200, 1000);
+		public static Size Android => new Size(1000, 1200);
```

Comment:
Here, instead of using a `magic number` can we use the Device Width? that way we can make sure that we will no renderer out the bounds of the screen... And, maybe, half of Device Height. What do you think?

Reply:
Good idea I was also wondering why it is initially hard-codeded in the sample.
using Essentials something like looks nice, what do you think ?

```
		public static Size Android
		{
			get
			{
				var width = 0.9 * DeviceDisplay.MainDisplayInfo.Width;
				var height = 0.6 * DeviceDisplay.MainDisplayInfo.Height;
				return new Size(width, height);
			}
		}
```

Reply:
I like that second one (: